### PR TITLE
Allowed origin header should be a single value, not a list

### PR DESCRIPTION
**Pull request by herojan**
If the backend sets the `Access-Control-Allow-Origin` header, the current corsOrigin filter appends another value. This pr makes it override the value instead.

Example chrome error from having multiple values in the allowed origin header: 

```
Access to XMLHttpRequest at 'https://4ce88b85.ngrok.io' from origin 'https://59298408.ngrok.io' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header contains multiple values 'https://nsns.ngrok.io, https://59298408.ngrok.io', but only one is allowed.
```

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

File: filters/cors/cors.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func (a filter) Response(ctx filters.FilterContext) {
 	}
 	for _, o := range a.allowedOrigins {
 		if o == origin {
-			ctx.Response().Header.Add(allowOriginHeader, o)
+			ctx.Response().Header.Set(allowOriginHeader, o)
```
Comment (aryszka) [inline filters/cors/cors.go]: there's another case on line 29, too. I think that should be also changed to .Set().

**Comment by herojan** [comment]
Changed that @aryszka

**Comment by mikkeloscar** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
thx @herojan !

**Pull request merged**